content/en/docs/concepts/services-networking/service.md

```diff
@@ -550,7 +551,7 @@ Traffic from the external load balancer is directed at the backend Pods. The clo
 
 For LoadBalancer type of Services, when there is more than one port defined, all
 ports must have the same protocol and the protocol must be one of `TCP`, `UDP`
-and `SCTP`.
+,and `SCTP`.
```

Comment:
Don't begin lines with punctuation.

Add a space after the comma.

Reply:
okay sure